packages/frameworks/src/frameworks.ts

```diff
@@ -2005,6 +2005,69 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Sanity',
```

Comment:
We'll want to use a different name, otherwise this will show up twice in the UI with no way to differentiate between the two.